pkg/apis/certmanager/v1alpha2/types_certificate.go

```diff
@@ -72,6 +72,11 @@ const (
 // A valid Certificate requires at least one of a CommonName, DNSName, or
 // URISAN to be valid.
 type CertificateSpec struct {
+	// Full X509 name specification (https://golang.org/pkg/crypto/x509/pkix/#Name).
+	// If specified, overrides any other name elements below.
```

Comment:
This line can be removed - we shouldn't handle backwards compatibility like this as we have API conversions for that 😄